### DISCUSSION TITLE
Questions about dev's ideology

### DISCUSSION THREAD

**Discussion by aicynide** [dyphire/mpv-config#33] [Q&A] [answered]
90% of the world population are influenced by English language and civilised culture.
The way dev rejecting to use Co-pilot or other AI to translate this project seems like he's a pro-CCP or 50 cent army

**Comment by dyphire** [comment, marked as answer]
#15 #16 #25 #32

I have already discussed the issue of English support in the above questions, which is difficult to achieve just due to my limited English proficiency. 
At the same time, I never stopped anyone from implementing English support and open a PR for it.

The Co-pilot only translates code language to another code language and does not support translating comments and other content. So I don't understand what you're talking about.
Although ChatGPT-4 can be used for translation, it is not free. Are you willing to pay for it?

What I don't understand is that since you have such a strong need for English support, why are you not willing to spend time implementing it and just keep complaining and even blaming others.

In fact, English support has always been on my plan list. But currently, less than 50% of the progress has been completed, and a large number of configuration files need to be translated, resulting in slow progress.
![image](https://github.com/dyphire/mpv-config/assets/61936050/a5a6f63f-0816-44e5-80fb-4b878c935288)
![image](https://github.com/dyphire/mpv-config/assets/61936050/74fe1a68-4a00-43db-a92b-7edb284d2b59)
![image](https://github.com/dyphire/mpv-config/assets/61936050/5f442803-7276-42fb-816c-1053af44dc49)
But due to your malicious slander, this plan will be put on hold indefinitely.
